starknet_py/tests/e2e/docs/guide/test_executing_transactions.py

```diff
@@ -2,25 +2,14 @@
 
 
 @pytest.mark.asyncio
-async def test_multicall(account, map_contract):
-    # docs-contract-interface: start
-    invoke_result = await map_contract.functions["put"].invoke(
-        key=10, value=20, max_fee=int(1e16)
-    )
-
-    await invoke_result.wait_for_acceptance()
-    # docs-contract-interface: end
-
-    (value,) = await map_contract.functions["get"].call(key=10)
-    assert value == 20
-
-    # docs-execute: start
+async def test_executing_transactions(account, map_contract):
+    # docs: start
     call = map_contract.functions["put"].prepare(key=20, value=20)
```

Comment:
Can we change this snippet to show how to create a `Call` directly instead of using `map_contract`?